mtgjson5/providers/tcgplayer.py

```diff
@@ -256,8 +269,194 @@ def generate_today_price_dict(
 
         return dict(combined_listings)
 
+    def determine_mtgjson_sealed_product_category(
+        self, product_name: str
+    ) -> MtgjsonSealedProductCategory:
+        """
+        Best-effort to parse the product name and determine the sealed product category
+        :param product_name Name of the product from TCG
+        :return: category
+        """
+
+        # The order of the checks is important
+        if any(
+            tag in product_name
+            for tag in [
```

Comment:
Would it make sense to const-ify these lists across the board? Would simplify the code here and maybe let us lambda function this

Reply:
i don't think so? these are just random names that happened to be on tcg and could change at any moment

what did you have in mind?

Reply:
Just a general idea I was thinking

```python3
def helper_with_better_name(find_list, return_value_if_found):
    return value_if_found if any(x for x in find_list) else None
```

and just repeatedly call to that setup. Could also be a lambda
```python3
lambda x, y: return y if any(a for a in x) else None
```

Reply:
i might need an example for this ^^